content/docs/handling-events.md

```diff
@@ -1,72 +1,72 @@
 ---
 id: handling-events
-title: Handling Events
+title: Manipulando eventos
 permalink: docs/handling-events.html
 prev: state-and-lifecycle.html
 next: conditional-rendering.html
 redirect_from:
   - "docs/events-ko-KR.html"
 ---
 
-Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+Manipular eventos em elementos com React é muito semelhante ao tratamento de eventos em elementos no DOM. Existem algumas diferenças sintáticas:
 
-* React events are named using camelCase, rather than lowercase.
-* With JSX you pass a function as the event handler, rather than a string.
+* Eventos em React são nomeados usando camelCase ao invés de letras minúsculas.
+* Com o JSX você passa uma função como manipulador de eventos ao invés de um texto.
 
-For example, the HTML:
+For exemplo, com HTML:
 
 ```html
 <button onclick="activateLasers()">
-  Activate Lasers
+  Ativar lasers
 </button>
 ```
 
-is slightly different in React:
+É ligeiramente diferente com React:
 
 ```js{1}
 <button onClick={activateLasers}>
-  Activate Lasers
+  Ativar lasers
 </button>
 ```
 
-Another difference is that you cannot return `false` to prevent default behavior in React. You must call `preventDefault` explicitly. For example, with plain HTML, to prevent the default link behavior of opening a new page, you can write:
+Outra diferença é que você não pode retornar `false` para evitar o comportamento padrão no React. Você deve chamar `preventDefault` explícitamente. Por exemplo, com HTML simples, para evitar que um link abra uma nova página, você pode escrever:
 
 ```html
-<a href="#" onclick="console.log('The link was clicked.'); return false">
-  Click me
+<a href="#" onclick="console.log('O link foi clicado.'); return false">
+  Clique em mim
 </a>
 ```
 
-In React, this could instead be:
+Em React, isso poderia ser:
 
 ```js{2-5,8}
 function ActionLink() {
   function handleClick(e) {
     e.preventDefault();
-    console.log('The link was clicked.');
+    console.log('O link foi clicado.');
   }
 
   return (
     <a href="#" onClick={handleClick}>
-      Click me
+      Clique em mim
     </a>
   );
 }
 ```
 
-Here, `e` is a synthetic event. React defines these synthetic events according to the [W3C spec](https://www.w3.org/TR/DOM-Level-3-Events/), so you don't need to worry about cross-browser compatibility. See the [`SyntheticEvent`](/docs/events.html) reference guide to learn more.
+Aqui, "`e`" é um evento sintético. O React define esses eventos sintéticos de acordo com a [especificação W3C](https://www.w3.org/TR/DOM-Level-3-Events/), então, não precisa se preocupar com a compatibilidade entre navegadores. Veja a página [`SyntheticEvent`](/docs/events.html) para saber mais.
```

Comment:
Is this case `SyntheticEvent` will refer to a page, wouldn't be better translating this to use the Portuguese? Because the name will match with the Portuguese page's title

Reply:
I agree with @eduardomoroni